docs/content/preview/manage/upgrade-pg15.md

```diff
@@ -0,0 +1,222 @@
+
+DDLs are not allowed even in this phase. New features that require format changes will not be available until the upgrade is finalized. Also, you cannot perform another upgrade until you have completed the current one.
+
+If you are satisfied with the new version, proceed to [finalize the upgrade](#finalize-phase).
+
+If you encounter any issues, you can [roll back](#rollback-phase) the upgrade.
+
+## Finalize phase
+
+Use the v2.25.1.0 yb-admin tool to finalize the upgrade.
+
+You need to run the commands `finalize_ysql_major_version_catalog_upgrade`, `promote_auto_flags`, and `upgrade_ysql` in order, as follows:
+
+```sh
+yb-admin --master_addresses <master_addresses> finalize_ysql_major_version_catalog_upgrade
+
+yb-admin --master_addresses <master_addresses> promote_auto_flags
+
+yb-admin --master_addresses <master_addresses> upgrade_ysql
+```
+
+For example:
+
+```sh
+./bin/yb-admin --master_addresses 127.0.0.1:7100,127.0.0.2:7100,127.0.0.3:7100 finalize_ysql_major_version_catalog_upgrade
+```
+
+```output
+Finalize successful
+```
+
+```sh
+./bin/yb-admin --master_addresses 127.0.0.1:7100,127.0.0.2:7100,127.0.0.3:7100 promote_auto_flags
+```
+
+```output
+PromoteAutoFlags completed successfully
+New AutoFlags were promoted
+New config version: 2
+```
+
+```sh
+./bin/yb-admin --master_addresses 127.0.0.1:7100,127.0.0.2:7100,127.0.0.3:7100 upgrade_ysql
+```
+
+```output
+YSQL successfully upgraded to the latest version
+```
+
+## Rollback phase
+
+### Roll back YB-TServers to v2024.2.2.0
+
+[Roll back all YB-Tservers](../upgrade-deployment/#1-roll-back-yb-tservers) to v2024.2.2.0.
+
+### Roll back the catalog
+
+Use the v2.25.1.0 yb-admin command `rollback_ysql_major_version_upgrade` to roll back the YSQL catalog. Use the command as follows:
+
+```sh
+yb-admin --master_addresses <master_addresses> rollback_ysql_major_version_upgrade
+```
+
+For example:
+
+```sh
+./yugabyte-2.25.1.0/bin/yb-admin --master_addresses 127.0.0.1:7100,127.0.0.2:7100,127.0.0.3:7100 rollback_ysql_major_version_upgrade
+```
+
+```output
+Rollback successful
+```
+
+### Roll back YB-Masters to v2024.2.2.0
+
+[Roll back all YB-Masters](../upgrade-deployment/#2-roll-back-yb-masters) to v2024.2.2.0.
+
+## Post upgrade
+
+After the upgrade is done, you can re-enable expression pushdowns. For example:
+
+```sh
+./bin/yb-ts-cli set_flag --server_address 127.0.0.1:9100 --force ysql_yb_enable_expression_pushdown true
+
+./bin/yb-ts-cli set_flag --server_address 127.0.0.2:9100 --force ysql_yb_enable_expression_pushdown true
+
+./bin/yb-ts-cli set_flag --server_address 127.0.0.3:9100 --force ysql_yb_enable_expression_pushdown true
+```
+
+Run ANALYZE on your databases to get the most optimal query plans.
+
+## Limitations
+
+- No yugabyted and Yugabyte Anywhere support. {{<issue 24793>}}
```

Comment:
yugabyted is not available. Can we update this doc to add the.
The partial steps are in https://yugabyte.slack.com/files/URNU51UH1/F08G7403RFB/pg15_upgrade_oss.sh

https://yugabyte.slack.com/archives/C07DG9B7C3T/p1741021856125019

Reply:
Lets not mention YBA. This is the OSS doc